cwms-data-api/src/main/java/cwms/cda/formatters/Formats.java

```diff
@@ -226,14 +227,29 @@ public static ContentType parseQueryParam(String queryParam) {
 
 
     public static ContentType parseHeader(String header) {
+        return parseHeader(header, null);
+    }
+
+    public static ContentType parseHeader(String header, Class<? extends CwmsDTOBase> klass) {
```

Comment:
I know you have the County IT but Formats.parseHeader is public static - would be real easy to add a test of the Format.parseHeader mapping for County in either a CountyTest or in a FormatsTest unit test.